README.md

```diff
@@ -23,21 +23,21 @@ on:
 
 jobs:
   build:
-    name: Test
+    # name of the job, you can rename this as needed
+    name: Crafting Preview Link
     runs-on: [ubuntu-latest]
     env:
-      # This is required to post a comment to PR.
+      # This tells the action to use the token so that it can post a comment to PR.
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - name: generate-preview-url # name of this step, you can rename this as needed
-        uses: crafting-dev/sandbox-launch-action/@VERSION
+        uses: crafting-dev/sandbox-launch-action@0.4.1 # update the version as needed
```

Comment:
I suggest every time we create a new tag, we update this, one less replacement for user to do.